domains/page.go

```diff
@@ -0,0 +1,23 @@
+package domains
+
+// ExclusiveStartKeyPagedResponse is object to use when returning database records in paged format using Exclusive Start Key paging
+type ExclusiveStartKeyPagedResponse[content any, lastEvaluatedKeyType any] struct {
```

Comment:
**Genral comment** 🙈 

In Go, it's a common convention for type names to start with an uppercase letter
```suggestion
type ExclusiveStartKeyPagedResponse[Content any, LastEvaluatedKeyType any] struct {
```

Reply:
Ah all right thank you, changed to capital letter